### DISCUSSION TITLE
BUild 2728 - now Russian intervention campaign?

### DISCUSSION THREAD

**Discussion by hreich1** [Khopa/dcs_liberation#1517] [Q&A] [answered]
Why is this campaign incomapatibile with preview version 2728 ?

**Comment by DanAlbert** [comment, marked as answer]
https://github.com/dcs-liberation/dcs_liberation/wiki/Campaign-maintenance#retired-campaigns

Any campaign that uses scenery objectives (normal map buildings as strike targets) breaks every time a DCS update updates the map. That happened, and the campaign author isn't working on the campaign any more.

**Comment by hreich1** [comment]
Ok thanks
